<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>480</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/3202.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>11</y>
     <width>45</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>温度：</string>
   </property>
  </widget>
  <widget class="QLabel" name="lab_temp">
   <property name="geometry">
    <rect>
     <x>94</x>
     <y>11</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>142</width>
     <height>19</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>燃气：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lab_gas">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>142</width>
     <height>19</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="label_8">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>湿度：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lab_hum">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>40</y>
     <width>142</width>
     <height>19</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QLabel" name="label_10">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>PM2.5：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lab_pm">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>142</width>
     <height>19</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <widget class="QLabel" name="label_12">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>光照：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lab_ill">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_5">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>70</y>
     <width>142</width>
     <height>19</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_7">
    <item>
     <widget class="QLabel" name="label_14">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>气压：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lab_air">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>142</width>
     <height>19</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_8">
    <item>
     <widget class="QLabel" name="label_16">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>烟雾：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lab_smoke">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_7">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>100</y>
     <width>142</width>
     <height>19</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_9">
    <item>
     <widget class="QLabel" name="label_18">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Co2：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lab_co2">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>142</width>
     <height>19</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_10">
    <item>
     <widget class="QLabel" name="label_20">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>人体：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lab_inf">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>无人</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QRadioButton" name="radio_mode1">
   <property name="geometry">
    <rect>
     <x>41</x>
     <y>311</y>
     <width>90</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string>离家模式</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radio_mode2">
   <property name="geometry">
    <rect>
     <x>41</x>
     <y>339</y>
     <width>90</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string>夜间模式</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radio_mode3">
   <property name="geometry">
    <rect>
     <x>41</x>
     <y>367</y>
     <width>90</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string>白天模式</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radio_mode4">
   <property name="geometry">
    <rect>
     <x>41</x>
     <y>395</y>
     <width>90</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string>安防模式</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chk_log">
   <property name="geometry">
    <rect>
     <x>41</x>
     <y>423</y>
     <width>90</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string>打开日志</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_tv">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>300</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>打开电视</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_air">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>330</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>打开空调</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_rfid">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>360</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>无线开门</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_fan">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>390</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>打开风扇</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_lamp">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>420</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>打开射灯</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_warning">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>180</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/3204.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_22">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>350</y>
     <width>75</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>监测对象：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmb_max">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>380</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>温度</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>光照</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="led_max">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>420</y>
     <width>151</width>
     <height>27</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_tcp">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>380</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>连接服务器</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_server">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>420</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>监听</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_23">
   <property name="geometry">
    <rect>
     <x>441</x>
     <y>321</y>
     <width>73</width>
     <height>17</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>服务器IP：</string>
   </property>
  </widget>
  <widget class="QLabel" name="lab_ip">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>321</y>
     <width>120</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_9">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>350</y>
     <width>201</width>
     <height>19</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_12">
    <item>
     <widget class="QLabel" name="label_25">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>端口号：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lab_port">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_9">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>450</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_27">
   <property name="geometry">
    <rect>
     <x>631</x>
     <y>11</y>
     <width>60</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>串口号：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmb1">
   <property name="geometry">
    <rect>
     <x>704</x>
     <y>11</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>ttyUSB0</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ttyUSB1</string>
    </property>
   </item>
  </widget>
  <widget class="QWidget" name="layoutWidget_10">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>40</y>
     <width>160</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_14">
    <item>
     <widget class="QLabel" name="label_28">
      <property name="text">
       <string>波特率：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="cmb2">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <item>
       <property name="text">
        <string>9600</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>115200</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_11">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>70</y>
     <width>167</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_15">
    <item>
     <widget class="QLabel" name="label_29">
      <property name="text">
       <string>校验位：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="cmb3">
      <item>
       <property name="text">
        <string>NONE</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_12">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>100</y>
     <width>160</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_16">
    <item>
     <widget class="QLabel" name="label_30">
      <property name="text">
       <string>数据位：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="cmb4">
      <property name="currentIndex">
       <number>3</number>
      </property>
      <item>
       <property name="text">
        <string>5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>6</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>7</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>8</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btn_serial">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>140</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>打开串口</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_curtain">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>150</y>
     <width>191</width>
     <height>161</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/3203.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>10</y>
     <width>300</width>
     <height>300</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <widget class="QLabel" name="lab_draw">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>300</width>
      <height>220</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>图表</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>240</y>
      <width>30</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>温度</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>240</y>
      <width>30</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>光照</string>
    </property>
   </widget>
  </widget>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>lab_temp</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget_2</zorder>
  <zorder>layoutWidget_3</zorder>
  <zorder>layoutWidget_4</zorder>
  <zorder>layoutWidget_5</zorder>
  <zorder>layoutWidget_6</zorder>
  <zorder>layoutWidget_7</zorder>
  <zorder>layoutWidget_8</zorder>
  <zorder>radio_mode1</zorder>
  <zorder>radio_mode2</zorder>
  <zorder>radio_mode3</zorder>
  <zorder>radio_mode4</zorder>
  <zorder>chk_log</zorder>
  <zorder>btn_tv</zorder>
  <zorder>btn_air</zorder>
  <zorder>btn_rfid</zorder>
  <zorder>btn_fan</zorder>
  <zorder>btn_lamp</zorder>
  <zorder>btn_warning</zorder>
  <zorder>label_22</zorder>
  <zorder>cmb_max</zorder>
  <zorder>led_max</zorder>
  <zorder>btn_tcp</zorder>
  <zorder>btn_server</zorder>
  <zorder>label_23</zorder>
  <zorder>lab_ip</zorder>
  <zorder>layoutWidget_9</zorder>
  <zorder>pushButton_9</zorder>
  <zorder>label_27</zorder>
  <zorder>cmb1</zorder>
  <zorder>layoutWidget_10</zorder>
  <zorder>layoutWidget_11</zorder>
  <zorder>layoutWidget_12</zorder>
  <zorder>btn_curtain</zorder>
  <zorder>btn_serial</zorder>
  <zorder>widget</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
